Zend/zend_inheritance.c

```diff
@@ -2933,6 +2933,10 @@ ZEND_API zend_class_entry *zend_do_link_class(zend_class_entry *ce, zend_string
 		ZSTR_SET_CE_CACHE(ce->name, ce);
 	}
 
+	if (ce->ce_flags & ZEND_ACC_ENUM && ce->enum_backing_type != IS_UNDEF) {
+		zend_enum_build_backed_enum_table(ce);
+	}
```

Comment:
This should be higher up, closer to zend_verify_enum.

Reply:
The backing table is currently not part of the inheritance cache but since this is fatal anyway it probably doesn't matter much.